src/main/java/mate.academy.internetshop/controller/BucketController.java

```diff
@@ -18,26 +19,28 @@
     @Inject
     private static OrderService orderService;
 
-    private Long bucketId;
+    private Long userId;
+    private Bucket bucket;
```

Comment:
If we move this variables to method, we'll need to initialize them twice (second time in doPost method). May they stay here?

Reply:
No, it isn't good practice because this controller will not work correctly in multithreading environment

Reply:
Thanks, got it. Made these variables local ones